resources/styles/book-content/questions.less

```diff
@@ -0,0 +1,13 @@
+.ost-chapter-review {
+  counter-reset: question;
```

Comment:
You mean add the reset in the base instead each section? The question is do we want the question numbering to be continuous or to reset for each section (ie critical thinking, review questions etc). In bio, chapter 3.1, for example, you have several types of questions, in my opinion it makes more sense to reset the questions for each section.